Makd.mak

```diff
@@ -534,6 +547,16 @@ $O/test-%.stamp: $O/test-%
 	$(call exec,$< $(ITFLAGS),$<,run)
 	$Vtouch $@
 
+# Documentation rules
+###################
+.PHONY: doc
```

Comment:
Wait! This is being [redefined](https://github.com/sociomantic-tsunami/makd/blob/v1.4.x/Makd.mak#L569-L572). You should only do a:

```make
doc += $O/doc.stamp
```

Maybe define an alias for this particular type of doc, like `ddoc` or `doc-d` or something.